src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -146,4 +148,77 @@ public void testSaveAndLoadMessages() throws PersistentDataStoreException {
     Assert.assertEquals(contentTwo, resultMessageTwo.getContent());
     Assert.assertEquals(creationTwo, resultMessageTwo.getCreationTime());
   }
+
+  @Test
+  public void testSaveAndLoadEvents() throws PersistentDataStoreException {
+    UUID idOne = UUID.randomUUID();
```

Comment:
Can you use an array and a for loop to generate this test data? Something like (didn't check syntax, probably incorrect):
```
List<String> eventTypes = Arrays.asList("User", "About Me", "Conversation", "Message");
List<Event> events = new ArrayList<Event>();
// Generate test events
for (int 0; i < 4; ++i) {
  events.add(new Event(
    UUID.randomUUID(),
    eventTypes[i],
    Instant.ofEpochMilli(1000 * (i + 1)),
    Arrays.asList("user_" + i, "some more information for event #" + i)
  ));
}
// Write test events
for (Event e: events) {
  persistentDataStore.writeThrough(e);
}
```

Reply:
I'll update this, it's much cleaner.